src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Endpoint.cs

```diff
@@ -0,0 +1,211 @@
+using System.Data.Common;
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Public.API.Helpers;
+using Dapper;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.Get;
+
+public sealed class Endpoint(AquiferDbContext _dbContext, ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Get("/resources/collections/{code}");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+        Summary(s =>
+        {
+            s.Summary = "Gets a resource collection with language localization data for the given collection code.";
+            s.Description =
+                "Returns the resource collection matching the collection code including the count of resource items in each available language.";
+        });
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        var (invalidLanguageIds, invalidLanguageCodes, validLanguageIds) = await _cachingLanguageService.ValidateLanguageIdsOrCodesAsync(
+            req.LanguageIds,
+            req.LanguageCodes,
+            shouldRequireInput: false,
+            ct);
+
+        if (invalidLanguageIds is not [])
+        {
+            ThrowError(r => r.LanguageIds, $"Invalid 'Language Ids': \"{string.Join("\", \"", invalidLanguageIds)}\"");
+        }
+        if (invalidLanguageCodes is not [])
+        {
+            ThrowError(r => r.LanguageCodes, $"Invalid 'Language Codes': \"{string.Join("\", \"", invalidLanguageCodes)}\"");
+        }
+
+        var parentResource = await GetParentResourceAsync(_dbContext.Database.GetDbConnection(), req.Code, ct);
```

Comment:
What's the reason to pass `_dbContext.Database.GetDbConnection()` as a parameter here, since the private method has access to `_dbContext` anyway?

Reply:
I intended to only call `_dbContext.Database.GetDbConnection()` only once and then pass the retrieved connection to each of the methods that needs it.  I'll fix that.

If/when we refactor to inject a `SqlConnectionFactory` (which we'd need to create) then the EF `DbContext` won't be necessary for any classes that strictly use Dapper, only a `SqlConnection`.  The `DbContext` has DI magic going on right now that automatically opens and closes `SqlConnection`s under the hood for the lifetime of the DI class (the `Endpoint` in our case) but you can turn off that magic in configuration.  Dapper doesn't have that, so we'd need to manually open/close the connection.  Why move in that direction?  Doing this refactor would be necessary to put in a profiling connection in order to log the SQL from Dapper commands (we can create such a connection from the factory).  The C# would look something like this:
```
using (var dbConnection = await _sqlConnectionFactory.OpenConnectionAsync(ct))
{
    var results = await dbConnection.QueryAsyncWithRetries(...);

    // the Dispose at the end of the using automatically closes the connection
}
```

Interestingly, the `DbContext` can be attached to an existing open `DbConnection` if we want to intermix EF and Dapper when using manual connections, but I'm not sure if it would double log EF queries at that point if we tried to enable SQL profiling logging.